policy-controller/core/src/lib.rs

```diff
@@ -47,6 +47,12 @@ pub enum AuthorizationRef {
     AuthorizationPolicy(String),
 }
 
+#[derive(Clone, Debug, Eq, Hash, PartialEq, PartialOrd, Ord)]
+pub enum RouteRef {
+    HttpRoute(String),
+    Probe(String),
```

Comment:
Per my other comment, we need to include default routes, so this should will be more than just probes... Default routes will always have a static name. (Same change could be made to `AuthorizationPolicyRef`)

```suggestion
    Default(&'static str),
```